examples/04_fuzzy_joining.py

```diff
@@ -416,9 +399,12 @@
 from sklearn.model_selection import GridSearchCV
 
 # We will test four possible values of match_score:
-params = {"joiner__match_score": [0.2, 0.3, 0.4, 0.5]}
+params = {
+    f"{joiner}__match_score": [0.2, 0.9]
+    for joiner in ["gdppc", "life_exp", "legal_rights"]
+}
 
-grid = GridSearchCV(pipeline, param_grid=params)
+grid = GridSearchCV(pipeline, param_grid=params, error_score="raise")
```

Comment:
Why this change?

Reply:
good catch, I'll remove it. I added it because otherwise the error that happened with the columntransformer we discussed this morning would get silently swallowed, so I added the `error_score` to make sure I could trigger it